### PR TITLE
Feature/update font style unit tests

### PR DESCRIPTION
**Pull request by ehm011**
## Description

Organized unit tests for FontStyle.vue, and added tests for default values.
## Related Issue or Trello Card

[Trello card](http://www.example.com/)

## Motivation and Context

Update units for thoroughness and best practices  
## How Has This Been Tested?

- [ ] Unit test
- [ ] NightWatch test
- [ ] Coverage testing
- [ ] Manual testing

## Types of changes

- [ ] New feature (non-breaking change which adds functionality)
- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist:

- [ ] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] I have added tests to cover my changes.
- [ ] All new and existing tests passed.

### CODE REVIEW

File: test/unit/specs/FontStyle.spec.js [hunk 1]
```diff
@@ -1,38 +1,42 @@
...
-describe('FontStyle.vue', () => {
-  it('Font Colors should be 11', () => {
-    const Constructor = Vue.extend(FontStyle)
-    const vm = new Constructor().$mount()
-    expect(vm.$data.fontColors.length).to.equal(11)
+describe('FontStyle.vue', function () {
+  let Constructor, vm
+
+  before(function () {
+    Constructor = Vue.extend(FontStyle)
```
Comment (KingMackle) [inline test/unit/specs/FontStyle.spec.js]: The `Constructor` does't need to be hoisted out into the scope of the spec, only the `vm` needs to

File: test/unit/specs/FontStyle.spec.js [hunk 2]
```diff
@@ -1,38 +1,42 @@
...
-})
 
-// Test number of font styles showing
-describe('FontStyle.vue', () => {
-  it('Font Styles should be 3', () => {
-    const Constructor = Vue.extend(FontStyle)
-    const vm = new Constructor().$mount()
-    expect(vm.$data.fontStyles.length).to.equal(3)
+  it('should have correct colors', function () {
+    let expectedColors = ['Blue', 'Red', 'Green', 'Yellow', 'Purple', 'Pink', 'Teal', 'Lime', 'Orange', 'Brown', 'Grey']
```
Comment (KingMackle) [inline test/unit/specs/FontStyle.spec.js]: Can we hoist line 20, 26, 32, and 38 to the scope of the spec file iteself

Comment (ehm011) [reply test/unit/specs/FontStyle.spec.js]: Done

File: test/unit/specs/FontStyle.spec.js (hunk 1 above)
Comment (ehm011) [reply test/unit/specs/FontStyle.spec.js]: done

**Pull request merged**